go/inst/instance_topology.go

```diff
@@ -511,13 +511,13 @@ func MoveBelow(instanceKey, siblingKey *InstanceKey) (*Instance, error) {
 	log.Infof("Will move %+v below %+v", instanceKey, siblingKey)
 
 	if maintenanceToken, merr := BeginMaintenance(instanceKey, GetMaintenanceOwner(), fmt.Sprintf("move below %+v", *siblingKey)); merr != nil {
-		err = fmt.Errorf("Cannot begin maintenance on %+v", *instanceKey)
+		err = fmt.Errorf("Cannot begin maintenance on %+v: merr", *instanceKey, merr)
```

Comment:
You probably meant this:
```suggestion
		err = fmt.Errorf("Cannot begin maintenance on %+v: %v", *instanceKey, merr)
```

Reply:
yep, my bad. I just wanted to see if this made sense. will fix.

Reply:
So I think patch should be good.